package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "heroku-pg-extras",
   "description": "A Heroku CLI plugin for awesome pg:* commands that are also great and fun and super.",
-  "version": "1.0.11",
+  "version": "1.0.12",
```

Comment:
so this version bump doesn't magically work via code, releasing a new version is documented in https://devcenter.heroku.com/articles/developing-cli-plugins#releasing-plugins; so i wouldn't touch the version here, but go through this devcenter article's instructions after merging to create a new release